### PR TITLE
Xvm values support

### PR DESCRIPTION
**Pull request by balexandre**
removed playerStats call as it's no longer supported by WoT API;
support for 2 different expected values origin (XVM and WnEfficiency);
expected values can now be easily dumped into a file;

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -84,23 +73,52 @@ angular.module('wotApp', ['wotServices.wn8'])
 		.success(function(data, status, headers, config) {
 			$scope.playerTanks = data.data[$scope.accountId];
 
+			//console.log('playerTanks', JSON.stringify($scope.playerTanks));
```
Comment (arthurhenning) [inline app.js]: Please remove the commented lines.

**Review by arthurhenning — COMMENTED** [review]
The commented lines should be removed, the rest looks ok. Has it been tested?

File: app.js (hunk 1 above)
Comment (balexandre) [reply app.js]: will do, was actually to see how many tanks we no longer use for stats as they were removed from the API.

And yes I did test, you will need to push to `gh-pages` as well so it has the latest version :)

Comment (balexandre) [reply app.js]: here's a test version: http://balexandre.github.io/WotStats/ of this branch

**Comment by balexandre** [comment]
all done...

test version on my fork page: http://balexandre.github.io/WotStats/

**Review by arthurhenning — APPROVED** [review]
Awesome job, Bruno. Highly appreciated!
Any further ideas for improvement are more than welcome!

**Pull request merged**